dev-itpro/administration/administration-center-api.md

```diff
@@ -27,7 +27,99 @@ For more information about administrative capabilities, see [The Business Centra
 
 The [!INCLUDE[prodadmincenter](../developer/includes/prodadmincenter.md)] API is located at the following URL: https://api.businesscentral.dynamics.com.
 
-## <a name="azuread"></a>Setting up Azure Active Directory (Azure AD) based authentication
+## Authenticate using service-to-service AAD Apps ([Client Credentials Flow](/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow))
+The [!INCLUDE[prodadmincenter](../developer/includes/prodadmincenter.md)] API supports authentication using AAD Apps.
```

Comment:
maybe we need to point out this is supported from 20.0+ only? Technically for some calls we support ANY platform version, but for these that need calls to platform permissions check, it must be 20.0+